### PR TITLE
Add CODEOWNERS entries for Scanner

### PR DESCRIPTION
**Pull request by RTann**
## Description

Scanner is migrating into this repo, so this PR adds a few CODEOWNERS entries for the team

## Checklist
- [x] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

## Testing Performed

Nah

### CODE REVIEW

**Review by vjwilson — APPROVED** [review]
Grammatically correct at last

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -30,3 +30,15 @@ pkg/postgres/**/*   @stackrox/data-shepherds
 tests/upgrade/*     @stackrox/data-shepherds
 
 operator/**/* @stackrox/draco
+
+# Scanner team's responsibilities include anything related to the scanner itself and scanning utilities
+# such as vulnerability uploading and image integrations.
+/central/scannerdefinitions/                          @stackrox/scanner
```
Comment (jvdm) [inline .github/CODEOWNERS:36]: `/central/enrichment`, `/pkg/nodes`, `/pkg/images`?

Comment (RTann) [reply .github/CODEOWNERS:36]: I'll add `pkg/nodes` and `pkg/images/enricher`. I don't think we own `central/enrichment` and `pkg/images` has other things which we don't own

Comment (jvdm) [reply .github/CODEOWNERS:36]: FYI, looking at the code it seems like `central/enrichment` has logic that manages integrations and their usage for both images (through deployments) and nodes. Also image metadata caching is initialized there. Also "blame" points out the usual contributors to Scanner (including you). Fine to keep it out, though, lots of intersection. If you want to keep out, we can always add it later if needed. Thanks.

**Review by jvdm — APPROVED** [review]
LGTM. Feel free to keep `/central/enrichment` out of it for now.

File: .github/CODEOWNERS (hunk 1 above)
Comment (RTann) [reply .github/CODEOWNERS:36]: Let's keep it out for now, and we can always add it, if we find it is necessary later

**Pull request merged**